web/custom-plugins/forecast-of-response/src/components/Comments.tsx

```diff
@@ -49,7 +71,81 @@ export const CompactCommentView = ({ comment }: ICompactCommentView) => {
       <div style={{ borderBottom: 'black 1px solid', width: 'fit-content' }}>
         {comment.operation}
       </div>
-      <MessageWrapper>{comment.message}</MessageWrapper>
+      <CompactMessageWrapper>{comment.message}</CompactMessageWrapper>
+    </CompactCommentWrapper>
+  )
+}
+
+export const CommentView = (props: { comment: TComment; even: boolean }) => {
+  const { comment, even } = props
+  const uniqueAuthorColor = colorFromString(comment.author)
+  return (
+    <CommentWrapper color={uniqueAuthorColor} even={even}>
+      <CommentHeaderWrapper>
+        <IconWrapper color={uniqueAuthorColor}>&#9679;</IconWrapper>
+        <b>{comment.author}</b>
+      </CommentHeaderWrapper>
+      <b style={{ width: 'fit-content' }}>
+        {new Date(comment.date).toLocaleString()}
+      </b>
+      <div style={{ borderBottom: 'black 1px solid', width: 'fit-content' }}>
+        {comment.operation}
+      </div>
+      <div>{comment.message}</div>
     </CommentWrapper>
   )
 }
+
+const InputWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin: 15px;
+`
+export const CommentInput = (props: { operationId: string }) => {
+  const { operationId } = props
+  const [message, setMessage] = useState<string>('')
+  const { token, userData } = useContext(AuthContext)
+  const dmssAPI = new DmssAPI(token)
+
+  function handlePost() {
+    // TODO: When we can import model contained data, remove name from Comment
+    const commentName = crypto.randomUUID()
+    dmssAPI.generatedDmssApi.explorerAddToParent({
```

Comment:
When i tested locally i got a 400 Bad request error from this function: "BadRequestException: Attribute not specified on parent".